packages/ubuntu_provision/lib/src/widgets/themed_list_tile.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:yaru/yaru.dart';
+
+class ThemedListTile extends StatefulWidget {
+  const ThemedListTile({
+    required this.valueKey,
```

Comment:
We don't need a separate `valueKey` here, and can just use the `key` instead. The `ListTile` further down the tree doesn't need a `key` if we have one here